src/main/java/Duckie.java

```diff
@@ -0,0 +1,47 @@
+package main.java;
+
+
+import main.java.classes.*;
+import main.java.command.Command;
+
+import java.io.IOException;
+
+public class Duckie {
+    private Storage storage;
+    private TaskList lst;
```

Comment:
Would it be better to rename "lst" to "list" here? Might be more understandable in the areas where it is used.